### PR TITLE
Migrate continuous integration from travis to github actions

### PR DESCRIPTION
**Pull request by dr0i**
- add github-actions' build.yml
- remove redundant dependency from pom.xml
- use https for sbt resolving typesafe server
- add maven central to resolvers for sbt
- switch status badge to github-actions
- remove travis build

See https://github.com/hbz/lobid-resources/issues/1103.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      run: |
+        git clone https://github.com/metafacture/metafacture-core.git
+        cd metafacture-core
+        ./gradlew install
+    - name: Build with Maven
+      run: mvn --batch-mode --update-snapshots verify
+    - name: Build with play
+      run: |
+        cd web
+        wget http://downloads.typesafe.com/typesafe-activator/${{env.ACTIVATOR_VERSION}}/typesafe-activator-${ACTIVATOR_VERSION}-minimal.zip ;unzip typesafe-activator-${{env.ACTIVATOR_VERSION}}-minimal.zip
```
Comment (katauber) [inline .github/workflows/build.yml:25]: I don't understand why one time `${{env.ACTIVATOR_VERSION}}` and one time `${ACTIVATOR_VERSION}` is used.

**Review by katauber — COMMENTED** [review]
That looks good for me. There's only one question in build.yml.

File: .github/workflows/build.yml (hunk 1 above)
Comment (dr0i) [reply .github/workflows/build.yml]: Good catch!
I wonder why it worked nonetheless (https://github.com/hbz/lobid-resources/runs/1707357417?check_suite_focus=true#step:6:3) - however, fixed this with last commit.

**Review by fsteeg — APPROVED** [review]

**Review by katauber — COMMENTED** [review]
+1

**Comment by dr0i** [comment]
Deployed, closed.

**Pull request merged**